documentation/source/conf.py

```diff
@@ -141,7 +142,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+#html_static_path = ['_static']
```

Comment:
This change doesn't seem to be related to this PR? If it isn't please remove it from this commit and/or open a new PR.

Reply:
Alright,  should be fine now.